project_proposal/tex/mainpage.tex

```diff
@@ -0,0 +1,376 @@
+The system must be able to find the optimal production speed for each product type, based on an error simulation and the appertaining graph upon which the error simulation is built.
+
+\subsection{List of requirements}
+\begin{table}[H]
+    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{1cm}|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|}
+        \hline
+        \textbf{ID} & \textbf{Name} & \textbf{Description} \\
+        \hline
+        R01 & Control production line & Start/stop production line \\
+        \hline
+        R02 & Monitor production & Monitor production (collect data and store said data) \\
+        \hline
+        R03 & Administer batches & Keep track of batches (batch ID) \\
+        \hline
+        R04 & Store batch info & Collect various data associated with current batch number from machine \\
+        \hline
+        R05 & Batch ID &Collect various data associated with current batch number from machine \\
+        \hline
+        R06 & Live data & Monitor and display live relevant data from the machine \\
+        \hline
+        R07 & Batch report & Produce a batch report (PDF/dashboard style format) \\
+        \hline
+        R08 & Documentation & Documentation must contain an illustration that defines the different components in the setup in relation to the ISA88^1213 Part 1 Physical Hierarchy model 	\\
+        \hline
+        R09 & Visualization & Visualization that can be accessed and used to display the production data                                                                                  	\\
+        \hline
+        R10 & OEE & Collect necessary data from the machine and calculate the OEE. OEE must be available to be displayed by the system                                          	\\
+        \hline
+        R11 & Estimate error function  	& Estimate the error function associated with the products                                                                                                    	\\
+        \hline
+        R12 & Optimal Production speed 	& Estimate the optimal production speed for each product type                                                                                                 	\\
+        \hline
+    \end{tabularx}
+    \caption{List of requirements} 
+    \label{table:requirements}
+\end{table} 
+
+\subsection{Prioritization of requirements}
+The group has decided that at the current state, every requirement is a must have.
+A further prioritization of requirements will occur, when we split the requirements into issues.
+
+\subsection{Use Case Diagram}
```

Comment:
The diagram is not under this section but moved down to the page under. Is it fixable?

Reply:
this should be fixed in the new version